### PR TITLE
Update ph_authors.yml

### LINKED ISSUE #2993: Preparing a small update to `ph_authors.yml`
I'm preparing a small update to `ph_authors.yml`:

- Updating the name, bio + affiliation of Nicolás Llano Linares.

### PR DESCRIPTION
**Pull request by anisa-hawes**
Update name, institution, affiliation + bio of Nicolás Llano Linares.

Closes #2993 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Assistant @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

### CODE REVIEW

**Comment by anisa-hawes** [comment]
Hello @jenniferisasi,

When I was checking the YAML header of generadores-aventura, I noticed that Nicolás's name is given as Nicolás Llano Linares. At the moment, in our `ph_authors.yml` file here on Jekyll, it is written as Nicolás Llano. As you know, these two need to match exactly.

At the same time, Nicolás asked me to update his affiliation and bio. He has provided me with the ES and PT translations 🙂 

--

I don't _think_ there is anywhere on the website where Nicolás's full bio appears in French, but I would be grateful if @marie-flesch might have a moment to double-check this translation?:

**FR:** Nicolás Llano Linares est docteur en sciences de la communication (Universidade de São Paulo, Brésil) et est professeur à l’École de la communication, des médias et de l’information, Fundação Getúlio Vargas, Brésil.

Note: Although this hasn’t been applied consistently across the Project Team bios (I am planning to initiate a review and edit soon), I think we can agree that we don’t translate institution names. Fundação Getúlio Vargas or University College London are the names of those institutions wherever we are in the world, and whichever language we speak. So, we can just translate the words and countries.

Thank you, 
Anisa

File: _data/ph_authors.yml [hunk 1]
```diff
@@ -2292,32 +2292,32 @@
...
   affiliation:
     en: |
-      Brazilian Institute of Education, Development and Research, Brazil
+      School of Communication, Media and Information, Fundação Getúlio Vargas, Brazil
     es: |
-      Instituto Brasilero de Enseñanza, Desarrollo e Investigación, Brasil
+      Escuela de Comunicación, Medios e Información, Fundação Getúlio Vargas, Brasil
     fr: |
-      Institut Brésilien d'enseignement, de développement et de recherche, Brésil
+      l’École de la communication, des médias et de l’information, Fundação Getúlio Vargas, Brésil
```
Comment (marie-flesch) [inline _data/ph_authors.yml]: remove "l'" : 
"École de la communication, des médias et de l’information, Fundação Getúlio Vargas, Brésil"

**Review by marie-flesch — COMMENTED** [review]
just a small change in affiliation

File: _data/ph_authors.yml (hunk 1 above)
Comment (anisa-hawes) [reply _data/ph_authors.yml]: Thank you, @marie-flesch. Done.

**Review by jenniferisasi — APPROVED** [review]

**Pull request merged**